src/main/java/androidx/text/Html.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.text
+
+import android.os.Build
+import android.text.Html
+import android.text.Html.ImageGetter
+import android.text.Html.TagHandler
+import android.text.Spanned
+
+/**
+ * Returns a new [Spanned] from [CharSequence] HTML.
+ *
+ * In Android 4.4, additional options were introduced to allow for more
```

Comment:
This does not match the implementation. Android N was 7.0, not 4.4.

Reply:
Fixed. Bad copy-pasta.